packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.html

```diff
@@ -0,0 +1,31 @@
+<form novalidate name="upgrade-form">
+    <oui-modal
+        data-heading="{{ 'dedicated_server_dashboard_upgrade_title_' + $ctrl.selectedUpgrade | translate }}"
+        data-primary-action="$ctrl.upgrade()"
+        data-primary-label="{{:: 'dedicated_server_dashboard_upgrade_submit_label' | translate }}"
+        data-primary-disabled="$ctrl.isLoading || !$ctrl.issueTypeDescription.trim()"
```

Comment:
`$ctrl.isLoading` is not necessary here, as it is used on `data-loading`, which will disable the whole modal if the variable is truthy